quickstarts/microsoft.databricks/databricks-workspace/main.bicep

```diff
@@ -0,0 +1,40 @@
+@description('Specifies whether to deploy Azure Databricks workspace with Secure Cluster Connectivity (No Public IP) enabled or not')
+param disablePublicIp bool = false
+
+@description('The name of the Azure Databricks workspace to create.')
+param workspaceName string
+
+@allowed([
+  'standard'
+  'premium'
+])
+@description('The pricing tier of workspace.')
+param pricingTier string = 'premium'
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+var managedResourceGroupName = 'databricks-rg-${workspaceName}-${uniqueString(workspaceName, resourceGroup().id)}'
+
+resource ws 'Microsoft.Databricks/workspaces@2018-04-01' = {
+  name: workspaceName
+  location: location
+  sku: {
+    name: pricingTier
+  }
+  properties: {
+    managedResourceGroupId: managedResourceGroup.id
+    parameters: {
+      enableNoPublicIp: {
+        value: disablePublicIp
+      }
+    }
+  }
+}
+
+resource managedResourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' existing = {
```

Comment:
Is this creating managed Resource group. If yes, we don't need to create managed resource group

Reply:
@msaranga Thanks for reviewing.  No, the 'existing' keyword indicates it's just saying it already exists (and they we can get its .id from it).